jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -1640,6 +1640,48 @@ void java_bytecode_parsert::rclass_attribute(classt &parsed_class)
     parsed_class.attribute_bootstrapmethods_read = true;
     read_bootstrapmethods_entry(parsed_class);
   }
+  else if(attribute_name == "InnerClasses")
+  {
+    u2 number_of_classes = read_u2();
```

Comment:
You might consider pulling this out into a `read_innerclass_attribute` which would allow you to document when you'd expect to see this and an overview of the info we're getting out of it, as I can't get my head around when this attribute appears and what we find in it, despite reading the spec.

Reply:
The spec is not great, agreed.